asmcomp/comballoc.ml

```diff
@@ -20,9 +20,9 @@ open Mach
 type allocation_state =
     No_alloc                            (* no allocation is pending *)
   | Pending_alloc of Reg.t * int        (* an allocation is pending *)
```

Comment:
This should maybe use an inline record in order to give a proper name to the "int".  In fact, maybe the names could be sufficiently good that the comment can then be removed.